src/services/option-services.ts

```
@@ -30,28 +39,33 @@ class OptionService {
   static async getNoticeItemList() {
     try {
       const noticeList = await this.getNoticesBase();
-      return (noticeList ?? []) as Notice[];
+      return noticeList ?? [];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)